### ISSUE TITLE
Remove `javax.transaction` in MANIFEST.MF

### ISSUE THREAD

**Issue by hussainnm** [eclipse-ee4j/ejb-api#80]
The `MANIFEST.MF` of **jakarta.ejb-api-4.0.0-RC1.jar** still has reference to _javax.transaction_

```
Import-Package: jakarta.ejb;version="4.0.0",jakarta.ejb.embeddable;ver
 sion="4.0.0",jakarta.ejb.spi;version="4.0.0",jakarta.enterprise.conte
 xt,jakarta.enterprise.util,jakarta.interceptor,jakarta.transaction;ve
 rsion="4.0.0",javax.naming,javax.transaction;version="[1.3,2.0)",java
 x.transaction.xa
```

**Comment by tkburroughs** [comment]
javax.transaction must be in the import, as it is referenced by the classes in the jar.  It should not be "exported".
The jar should end up with imports/exports like this:

Export-Package: jakarta.ejb;version="4.0";uses:="jakarta.transaction",jakarta.ejb.embeddable;version="4.0";uses:="jakarta.naming",jakarta.ejb.spi;version="4.0";uses:="jakarta.ejb,jakarta.ejb.embeddable"
Import-Package: jakarta.naming,jakarta.transaction;version="2.0.0"

**Comment by hussainnm** [comment]
Current:
 ```
Import-Package: jakarta.ejb;version="4.0.0",jakarta.ejb.embeddable;ver
 sion="4.0.0",jakarta.ejb.spi;version="4.0.0",jakarta.transaction;vers
 ion="2.0",javax.naming,javax.transaction;version="[1.3,2.0)"
Tool: Bnd-0.0.255
Implementation-Vendor: Eclipse Foundation
Export-Package: jakarta.ejb.spi;uses:="jakarta.ejb,jakarta.ejb.embedda
 ble";version="4.0.0",jakarta.ejb;uses:="jakarta.transaction";version=
 "4.0.0",jakarta.ejb.embeddable;uses:="jakarta.ejb.spi,jakarta.ejb,jav
 ax.naming";version="4.0.0" 
```
Proposed:
``` 
Import-Package: jakarta.ejb;version="4.0.0",jakarta.ejb.embeddable;ver
 sion="4.0.0",jakarta.ejb.spi;version="4.0.0",jakarta.transaction;vers
 ion="2.0",javax.naming
Tool: Bnd-0.0.255
Implementation-Vendor: Eclipse Foundation
Export-Package: jakarta.ejb.spi;uses:="jakarta.ejb,jakarta.ejb.embedda
 ble";version="4.0.0",jakarta.ejb;uses:="jakarta.transaction";version=
 "4.0.0",jakarta.ejb.embeddable;uses:="jakarta.ejb.spi,jakarta.ejb,jav
 ax.naming";version="4.0.0" 
```

In the current build both jakarta.transaction;version="2.0" and javax.transaction;version="[1.3,2.0)" are added to Import-Package.

Since we have migrated javax.transaction to jakarta.transaction, this no longer needs to be added in MANIFEST.MF.

**Comment by kwsutter** [comment]
Just a reminder that the javax.transaction.xa package is now part of the JDK.  The rest of javax.transaction is part of Jakarta EE and will be renamed to jakarta.transaction during this transition.  Not sure if this affects the decision on the import or not, just wanted to remind people that not everything transactions-related is moving to the jakarta namespace...

**Comment by hussainnm** [comment]
This project imports only one API jakarta.transaction.UserTransaction from jakarta.transaction-api.jar. There are no imports from either javax.transaction or javax.transaction.xa.

The contents of MANIFEST.MF I have posted is from jakarta.ejb-api-4.0.0-RC1.jar. Here javax.transaction.xa is included in the Import-Package due to the regression of including jakarta.transaction package. This was fixed in #79 .

Now what remains is the javax.transaction which is added explicitly in pom.xml.
